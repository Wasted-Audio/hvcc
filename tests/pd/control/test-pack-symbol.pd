#N canvas 750 181 363 362 10;
#X obj 24 11 loadbang;
#X obj 24 200 pack;
#X obj 24 127 t b b b;
#X msg 24 167 10;
#X msg 53 167 100;
#X obj 170 126 t b b;
#X obj 98 211 pack f f;
#X obj 117 181 t b f;
#X msg 128 155 200;
#X obj 98 124 t b b;
#X msg 98 154 10;
#X obj 24 326 print;
#X obj 24 75 t b b b;
#X obj 24 39 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 170 211 pack 100 s;
#X msg 219 181 symbol bla;
#X connect 0 0 13 0;
#X connect 1 0 11 0;
#X connect 2 0 3 0;
#X connect 2 1 4 0;
#X connect 2 2 3 0;
#X connect 3 0 1 0;
#X connect 4 0 1 1;
#X connect 5 0 14 0;
#X connect 5 1 15 0;
#X connect 6 0 11 0;
#X connect 7 0 6 0;
#X connect 7 1 6 1;
#X connect 8 0 7 0;
#X connect 9 0 10 0;
#X connect 9 1 8 0;
#X connect 10 0 6 0;
#X connect 12 0 2 0;
#X connect 12 1 9 0;
#X connect 12 2 5 0;
#X connect 13 0 12 0;
#X connect 14 0 11 0;
#X connect 15 0 14 1;
