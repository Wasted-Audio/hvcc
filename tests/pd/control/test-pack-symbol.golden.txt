[@ 0.000] print: 100 bla
[@ 0.000] print: 0 200
[@ 0.000] print: 10 200
[@ 0.000] print: 10 0
[@ 0.000] print: 10 100
